cmd/analyze/main.go

```diff
@@ -206,3 +212,31 @@ func main() {
 		dynamicAnalysis(pkg)
 	}
 }
+
+func generateSSLCerts() error {
+	log.Debug("Generating SSLSplit CA certificates")
+	createSslCerts := exec.Command("openssl", "genrsa", "-out", "/proxy/certs/ca.pem", "2048")
+	createSslCerts.Dir = "/proxy/certs"
+	err := createSslCerts.Start()
+
+	if err != nil {
```

Comment:
Style nit: for statements like 
```
err := func()

if err != nil {
   ...
}
```
prefer the more compact style: 
```
if err := func(); err != nil {
    ...
}

There are a couple of places in this PR where I think this can improve conciseness